packages/client/src/pages/Forum/TopicConversationPage/ui/components/CommentList/components/ReactionBlock/store/helpers/filterUniqueReaction.ts

```diff
@@ -0,0 +1,7 @@
+import { TReactionItem } from '@root/pages/Forum/TopicConversationPage/store/types';
+
+export const filterUniqueReaction = (reactionData: TReactionItem[]) => {
+  const reactionNameList = reactionData.map(item => item.reactionName);
+
+  return Array.from(new Set(reactionNameList));
```

Comment:
[Предложение]
Можно в одну строчку и без переменной)

```suggestion
  return [...new Set(reactionData.map(item => item.reactionName))];
```

Reply:
Окей. В моем понимании разбивка по шагам повышает читаемость кода.

Reply:
Тут стоит обсудить с командой, где эти границы читаемости. 
Мне одинокого просто прочитать тут и одну строку, и разбитое на две, поэтому предложила объединить )

Reply:
Окей.